src/core/thread/mle_constants.hpp

```diff
@@ -79,8 +79,9 @@ enum
     kMinTimeoutDataPoll             = (OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD +
                            OPENTHREAD_CONFIG_FAILED_CHILD_TRANSMISSIONS * OPENTHREAD_CONFIG_RETX_POLL_PERIOD) /
                           1000, ///< Minimum timeout(s) for data poll
-    kMinTimeout = (kMinTimeoutKeepAlive >= kMinTimeoutDataPoll ? kMinTimeoutKeepAlive
+    kMinTimeout         = (kMinTimeoutKeepAlive >= kMinTimeoutDataPoll ? kMinTimeoutKeepAlive
                                                                : kMinTimeoutDataPoll), ///< Minimum timeout(s)
+    kChildTimeoutMargin = 10, ///< Margin for duplicate parent request
```

Comment:
Is that enough? Some rough calculation (neglecting CPU processing time) for transmission delay (without ACK) is:

`T_transmit_max ~= T_csma_max + T_frame_tx ~= 27ms + 4.25ms > 31.25ms`

I would recommend making a margin equal to 50ms then.